tests/unit/store/printMyBarcode/actions.spec.js

```diff
@@ -6,14 +6,23 @@ const formatDate = () => {
   return `${dd}-${mmm}-${yyyy.slice(2)}`
 }
 
+const getSuffix = (suffix, barcode) => {
+  return suffix ? barcode.slice(-5) : ''
```

Comment:
is the suffix always the same length?

Reply:
If not, might be better to get the text after the last hyphen if suffix always has a hyphen

Reply:
It is agreed that the suffix will be four letters(UPAB) plus one number representing the number of labels(1-9) - UPAB1